cmd/envbuilder/main.go

```diff
@@ -23,15 +23,14 @@ import (
 )
 
 func main() {
-	root := &cobra.Command{
-		Use: "envbuilder",
+	config := envbuilder.Config{}
+	cmd := serpent.Command{
+		Use:     "envbuilder",
+		Options: config.Options(),
 		// Hide usage because we don't want to show the
 		// "envbuilder [command] --help" output on error.
-		SilenceUsage:  true,
-		SilenceErrors: true,
-		RunE: func(cmd *cobra.Command, args []string) error {
-			options := envbuilder.OptionsFromEnv(os.LookupEnv)
-
+		Hidden: true,
```

Comment:
Hidden: false?

Looks like in coder/serpent this property means something else: `// Hidden determines whether the command should be hidden from help.`

Reply:
Since this is one command CLI we don't want to show it in the help as an "option" but I'm not sure I'm 100% right. @ammario maybe help us.

Reply:
I don't think Hidden has any effect on the root command. It determines whether the command shows up as a subcommand to its parent during help, and there is no parent here.